### ISSUE TITLE
Incorrect Color Usage for Icons

### ISSUE THREAD

**Issue by simonfossom** [omnivore-app/omnivore#3355]
Issue: any kind of dark theming on webpage makes icons invisible because they contain color. 

Expected behavior: icons should get color from text (or some other way depending on design.

Environment: Chrome browser with [extension Dark Reader](https://darkreader.org/)

**Comment by jacksonh** [comment]
Is there a reason you need to use dark reader instead of our themes?

**Comment by simonfossom** [comment]
Yes, I want full black and not dark if I can have it :)

**Comment by jacksonh** [comment]
Got it. We actually do have full black but only the mobile clients use it. We should probably expose it on web. 

With dark reader, I guess I need to look into how it sets all these values. We have quite a few colors invovled in a theme though, so I'm worried it will be hard to get the two to play nicely together.

**Comment by simonfossom** [comment]
Yes. You can close this issue. The problem is in Dark Reader. It incorrectly adds color to the icons.

**Comment by jacksonh** [comment]
I'll just update the title, I know there have been a few people that had issues in the past related to dark reader.

**Comment by jacksonh** [comment]
Linking these: https://github.com/omnivore-app/omnivore/issues/3356
